flexget/plugins/urlrewrite_divxatope.py

```diff
@@ -8,24 +8,59 @@
 from flexget.utils import requests
 from flexget.utils.soup import get_soup
 
+from flexget.entry import Entry
+from flexget.utils.search import torrent_availability, normalize_unicode
+
 log = logging.getLogger('divxatope')
 
+CATEGORIES = {
```

Comment:
WIP. I want to finish when I have more time to test. At this version the plugin uses "all" by default.